functions/src/notifications/sendNotifications.ts

```diff
@@ -0,0 +1,73 @@
+import { InvitationDestination } from '../types/InvitationDestination'
+import { NotificationContent } from '../types/Notification'
+import { OVHCredentials } from '../types/OVHCredentials'
+import { SendGridEnv } from '../types/SendGridEnv'
+import { getAppEnv } from '../firebase/env'
+import { sendNotification } from './sendNotification'
+
+export interface SendNotificationsResult {
+    emailsSent: string[]
+    smssSent: string[]
+}
+
+export const sendNotifications = async (
+    destinations: InvitationDestination[],
+    notificationContent: NotificationContent,
+    ovhCredentials?: OVHCredentials,
+    sendGridEnv?: SendGridEnv
+): Promise<SendNotificationsResult> => {
+    // Just some quick prefilters
+    const emailDestinations = destinations.filter(
+        (dest) => dest.email && dest.email.length > 4
```

Comment:
may we add this [prefer-optional-chain](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md) linter rule ?

Reply:
I agree, you can open another PR with the rule :p

Reply:
not possible here I think

Reply:
you should be able to go with:

```js
(dest) => dest?.email?.length > 4
```

Reply:
cannot do